app/src/full/AndroidManifest.xml

```diff
@@ -2,7 +2,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
-    <uses-sdk tools:overrideLibrary="com.google.android.gms.threadnetwork" />
+    <uses-sdk tools:overrideLibrary="com.google.android.gms.threadnetwork,androidx.car.app.projected" />
+    <uses-permission android:name="com.google.android.gms.permission.CAR_FUEL" />
+    <uses-permission android:name="com.google.android.gms.permission.CAR_MILEAGE" />
```

Comment:
These should be moved over to the `main` manifest now :)

Reply:
Only the SDK override, permissions here are GMS permissions so only applicable to the full flavor

Reply:
> Only the SDK override, permissions here are GMS permissions so only applicable to the full flavor

they seem to be used for a few sensors though i thought

Reply:
permissions with GMS substring are related to the phone, for the car  you will have things like 
android.car.permission.CAR_SPEED are for cars.